nbcelltests/shared.py

```diff
@@ -152,9 +156,29 @@ def is_empty(source):
     return len(parsed.body) == 0
 
 
-def cell_injected_into_test(test_lines):
-    for test_line in test_lines:
-        if test_line.strip().startswith(r"%cell"):
+CELL_INJ_TOKEN = r"%cell"
+
+
+def source2lines(source):
+    return source.splitlines(keepends=True)
+
+
+def lines2source(lines):
+    return "".join(lines)
+
+
+def cell_inj_span(test_line):
```

Comment:
Ideally a docstring here would be useful. Also, expanding the function name would be helpful:
- `cell_inj_span`: Is this "cell inject span", i.e. is this code injecting something?
- `[get_]cell_injection_span`: Here it is clearer that we are getting the "injection span" (and hopefully the docstring explains what an injection span is).